test/testapps/SimpleMvc/Program.cs

```diff
@@ -5,6 +5,9 @@ public class Program
     {
         public static void Main(string[] args)
         {
+            // Just make sure we have a reference to the MvcShim
+            var t = typeof(Microsoft.AspNetCore.Mvc.IActionResult);
+            System.Console.WriteLine(t.FullName);
```

Comment:
This is a nice canary for tests. I want the build to fail creating the SimpleMvc assembly if something is wrong.